multimedia/gst1-plugins-base/patches/004-Synchronize-playback-support-using-alsa-start-at.patch

```diff
@@ -0,0 +1,3029 @@
+timestamps.
+
+Add property 'start-at' in alsasink for additional delay
+of the sink in nanoseconds to sync playback start in hardware.
+
+Change-Id: I1189aaa8e2d66f664afe89a63490b792ca928143
+Signed-off-by: Abhimanyu Vishwakarma <Abhimanyu.Vishwakarma@imgtec.com>
+Signed-off-by: Matin Momin <Matin.Momin@imgtec.com>
+Signed-off-by: Sagar Tikore <Sagar.Tikore@imgtec.com>
+Signed-off-by: Marcin Nowakowski <marcin.nowakowski@imgtec.com>
+Signed-off-by: Abhilash Tuse <Abhilash.Tuse@imgtec.com>
+---
+ ChangeLog                                          |  11 +
+ config.rpath                                       | 152 ++++++++------
+ configure.ac                                       |  10 +-
+ ext/alsa/gstalsasink.c                             | 188 ++++++++++++++++-
+ ext/alsa/gstalsasink.h                             |   3 +
+ ext/alsa/gstalsasrc.c                              |  78 ++++++-
+ ext/alsa/gstalsasrc.h                              |   2 +
+ gst-libs/Makefile.am                               |   3 +-
+ gst-libs/ext/Makefile.am                           |   9 +
+ gst-libs/ext/alsaclock/Makefile.am                 |  20 ++
+ gst-libs/ext/alsaclock/gstalsaclock.c              | 232 +++++++++++++++++++++
+ gst-libs/ext/alsaclock/gstalsaclock.h              |  76 +++++++
+ gst-libs/gst/audio/gstaudiobasesink.c              |  39 +++-
+ gst-libs/gst/audio/gstaudiobasesink.h              |   6 +
+ m4/gettext.m4                                      |  96 +++++----
+ m4/iconv.m4                                        | 187 ++++++++++++-----
+ m4/lib-ld.m4                                       |  87 ++++----
+ m4/lib-link.m4                                     | 138 ++++++++----
+ m4/lib-prefix.m4                                   |  91 +++++---
+ m4/nls.m4                                          |  13 +-
+ m4/po.m4                                           |  40 ++--
+ m4/progtest.m4                                     |  31 ++-
+ pkgconfig/Makefile.am                              |  15 ++
+ pkgconfig/gstreamer-alsaclock-uninstalled.pc.in    |  13 ++
+ pkgconfig/gstreamer-alsaclock.pc.in                |  11 +
+ pkgconfig/gstreamer-plugins-base-uninstalled.pc.in |   4 +-
+ pkgconfig/gstreamer-plugins-base.pc.in             |   2 +-
+ po/Makefile.in.in                                  |  88 ++++++--
+ po/Rules-quot                                      |  15 +-
+ 29 files changed, 1330 insertions(+), 330 deletions(-)
+ create mode 100644 gst-libs/ext/Makefile.am
+ create mode 100644 gst-libs/ext/alsaclock/Makefile.am
+ create mode 100644 gst-libs/ext/alsaclock/gstalsaclock.c
+ create mode 100644 gst-libs/ext/alsaclock/gstalsaclock.h
+ create mode 100644 pkgconfig/gstreamer-alsaclock-uninstalled.pc.in
+ create mode 100644 pkgconfig/gstreamer-alsaclock.pc.in
+
+diff --git a/ChangeLog b/ChangeLog
+index 4b624f5..829d9c9 100644
+--- a/ChangeLog
++++ b/ChangeLog
+@@ -1,3 +1,14 @@
++2016-11-22  gettextize  <bug-gnu-gettext@gnu.org>
```

Comment:
gettextize update has nothing to do with with adding playback support, so why is it added in this patch file?